### PR TITLE
Create ./generate.sh and ./embed.sh for flexible flow

### PR DESCRIPTION
**Pull request by omertuc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,31 @@ See https://github.com/openshift/enhancements/pull/565
 	- Boot the VM with that ISO
 - You can now monitor the progress using `make ssh` and `journalctl -f -u bootkube.service` or `kubectl --kubeconfig ./sno-workdir/auth/kubeconfig get clusterversion`
 
+# How to run - manual mode
+- Create a workdir for the installer - `mkdir sno-workdir`
+- Create an `install-config.yaml` in the sno-workdir. An example file can be found in `./install-config.yaml.template`
+- Download the ISO to the workdir `./download_live_iso.sh sno-workdir/base.iso`
+- Get an installer binary using `oc adm extract` or `./hack/build.sh`
```
Comment (omertuc) [inline README.md]: @eranco74 you might want to elaborate more here, feel free to edit. I wasn't sure what to write.

File: generate.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	echo "Please set INSTALLER_BIN"
+	exit 1
+fi
+
+if [ -z ${INSTALLER_WORKDIR+x} ]; then
+	echo "Please set INSTALLER_WORKDIR"
+	exit 1
+fi
+
+OPENSHIFT_INSTALL_EXPERIMENTAL_BOOTSTRAP_IN_PLACE=true \
```
Comment (eranco74) [inline generate.sh]: You can remove this line

File: embed.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+if [[ $(dirname $ISO_PATH) != $(dirname $IGNITION_PATH) ]]; then
+    echo $ISO_PATH and $IGNITION_PATH must be in the same directory
+    exit 1
+fi
+
+if [[ $(dirname $IGNITION_PATH) != $(dirname $OUTPUT_PATH) ]]; then
+    echo $IGNITION_PATH and $OUTPUT_PATH must be in the same directory
+    exit 1
+fi
```
Comment (eranco74) [inline embed.sh]: I think you can remove this and just add another volume mount to the `podman run` command, e.g.:
```
-v $(realpath $(dirname $ISO_PATH)):/data \
-v $(realpath $(dirname $IGNITION_PATH)):/ignition_data \
```
And update the `embed` param:
```
--ignition-file /ignition_data/$(basename $IGNITION_PATH)

File: README.md (hunk 1 above)
Comment (eranco74) [inline README.md]:
```suggestion
- Build the installer binary using `./hack/build.sh`
```

Comment (eranco74) [reply README.md]: `oc adm extract` isn't an option since we no longer have a release image with an up to date installer that can install single node.

**Pull request merged**